README.md

```diff
@@ -69,10 +69,15 @@ client = CompaniesHouse::Client.new(config)
 The client is configured by passing a hash to the constructor. The supported keys for this
 hash are:
 
-| Key         | Description |
-| ----------- | ----------- |
-| `:api_key`  | Required. The API key received after registration. |
-| `:endpoint` | Optional. Specifies the base URI for the API (e.g. if using a self-hosted version) |
+| Key                | Description |
+| -----------        | ----------- |
+| `:api_key`         | Required. The API key received after registration. |
+| `:endpoint`        | Optional. Specifies the base URI for the API (e.g. if using a self-hosted version) |
+| `:instrumentation` | Optional. Instruments the request/response (see Instrumentation for details) |
```

Comment:
There's a way we can ship this change without making a breaking API change (for users depending on notifications). How about this?

1. If `defined?(ActiveSupport)`, we automatically configure `:instrumentation`. This is consistent with other Rails gems that essentially autoconfigure if in a Rails environment ([example](https://github.com/yammer/circuitbox/blob/0c508f50f56531eb191438ed19c8d527df9a8e91/lib/circuitbox/configuration.rb#L26-L30)).
2. If you pass any other argument to `:instrumentation`, that is configured instead.

Reply:
lol perfect, `circuitbox` is using pretty much the same thing 😄 Will do that 🙏

Reply:
Do you prefer the name `Notifier` or `Instrumentation`?

Reply:
No real preference here, as long as it's documented!